cloudbuild.yaml

```diff
@@ -0,0 +1,21 @@
+# See https://cloud.google.com/cloud-build/docs/build-config
+
+# this must be specified in seconds. If omitted, defaults to 600s (10 mins)
+timeout: 7200s
+steps:
+  - name: 'gcr.io/k8s-testimages/gcb-docker-gcloud:v20210722-085d930'
+    entrypoint: bash
+    env:
+    - PROW_GIT_TAG=$_GIT_TAG
+    - PULL_BASE_REF=$_PULL_BASE_REF
+    - VERSION=$_PULL_BASE_REF
+    args:
+    - -c
+    - |
+      echo "Building/Pushing NPD containers"
+      make push-container
+substitutions:
+  # _GIT_TAG will be filled with a git-based tag for the image, of the form vYYYYMMDD-hash, and
+  # can be used as a substitution
+  _GIT_TAG: '12345'
```

Comment:
minor: is this '12345' a placeholder? better use 'PLACE_HOLDER' I guess.

Reply:
Agree